tests/flow/tests_common.py

```diff
@@ -90,56 +95,58 @@ def test_common_tensorset_error_replies(env):
     # ERR invalid value
     try:
         con.execute_command('AI.TENSORSET', 'z', 'INT32', 2, 'VALUES', 2, 'A')
+        env.assertFalse(True)
     except Exception as e:
         exception = e
         env.assertEqual(type(exception), redis.exceptions.ResponseError)
         env.assertEqual(exception.__str__(), "invalid value")
 
     try:
         con.execute_command('AI.TENSORSET', 1)
+        env.assertFalse(True)
     except Exception as e:
         exception = e
         env.assertEqual(type(exception), redis.exceptions.ResponseError)
 
     try:
         con.execute_command('AI.TENSORSET', 'y', 'FLOAT')
-    except Exception as e:
-        exception = e
-        env.assertEqual(type(exception), redis.exceptions.ResponseError)
-
-    try:
-        con.execute_command('AI.TENSORSET', 'y', 'FLOAT', '2')
```

Comment:
extra delete?

Reply:
The diff here isn't shown properly. 
I removed this one check:
con.execute_command('AI.TENSORSET', 'y', 'FLOAT', '2')
since it is a valid command (this test wasn't suppose to pass...)